quinn-udp/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quinn-udp"
-version = "0.7.0"
+version = "0.8.0-beta.1"
```

Comment:
Should this be 0.1.0 (since this is a new crate) or 0.8.0 (to match quinn/quinn-proto)?

Reply:
I vote for 0.1.0.

Reply:
I don't feel any particular need to artificially sync versions, especially as this is an impl detail for most users. It also seems unlikely that the breakage cadence will match that of the main public APIs in the long term. So, 0.1.0 seems fine.